cdc/owner/changefeed.go

```diff
@@ -597,13 +597,13 @@ LOOP2:
 	}
 	c.barriers.Update(finishBarrier, c.latestInfo.GetTargetTs())
 
-	f, err := filter.NewFilter(c.latestInfo.Config, "")
+	filter, err := filter.NewFilter(c.latestInfo.Config, "")
```

Comment:
`filter` is also the package name, so I renamed it to `f`.  `filter` looks good to me, but report warning by the IDE

Reply:
Use the package alias `pfilter`, PTAL